modules/document_repository/ajax/documentEditUpload.php

```diff
@@ -58,26 +60,61 @@
         $instrument = $_POST['instrument'] !== '' ? $_POST['instrument'] : null;
         $pscid      = $_POST['pscid']      !== '' ? $_POST['pscid'] : null;
         $visit      = $_POST['visit']      !== '' ? $_POST['visit'] : null;
-        $comments   = $_POST['comments']   !== '' ? $_POST['commnets'] : null;
+        $comments   = $_POST['comments']   !== '' ? $_POST['comments'] : null;
         $version    = $_POST['version']    !== '' ? $_POST['version'] : null;
+        $uuid       = uuid4();
 
         $fileSize = $_FILES["file"]["size"];
         $fileName = $_FILES["file"]["name"];
-        $fileType = end((explode(".", $fileName)));
+        $fileType = '';
+        // Handle retrieving the file type.
```

Comment:
This is better done using [`pathinfo()`](https://secure.php.net/manual/en/function.pathinfo.php)

It's also more accurate to say that you are grabbing the uploaded file's extension. Properly getting the file type would involve looking at the file signatures.